cmd/deploy/command_configuration_test.go

```diff
@@ -207,6 +207,18 @@ func Test_getStackServicesToDeploy(t *testing.T) {
 			},
 			expected: []string{},
 		},
+		{
+			name: "MultipleComposeInfo",
```

Comment:
There are 3 test cases with the same title. Could we rename them to make them more specific? What is the difference between each of them?